src/components/cl/basic/cl_basic_context.c

```diff
@@ -63,9 +63,13 @@ UCC_CLASS_CLEANUP_FUNC(ucc_cl_basic_context_t)
 
 UCC_CLASS_DEFINE(ucc_cl_basic_context_t, ucc_cl_context_t);
 
-ucc_status_t ucc_cl_basic_get_context_attr(const ucc_base_context_t *context, /* NOLINT */
-                                           ucc_base_attr_t *attr)             /* NOLINT */
+ucc_status_t
+ucc_cl_basic_get_context_attr(const ucc_base_context_t *context, /* NOLINT */
+                              ucc_base_ctx_attr_t      *attr)
 {
-    /* TODO */
-    return UCC_ERR_NOT_IMPLEMENTED;
+    if (attr->attr.mask & UCC_CONTEXT_ATTR_FIELD_CTX_ADDR_LEN) {
+        attr->attr.ctx_addr_len = 0;
+    }
+    // attr->attr.mask & UCC_CONTEXT_ATTR_FIELD_CTX_ADDR - nothing to do
```

Comment:
Minor: dead code? plz remove it.

Reply:
do you mean the comment? It explains why the address is not packed (CL BASIC has no address, at least right now). The ctx_addr_len setting to 0 is valid (not dead) code and must be there.

Reply:
Yes, the comment.